src/sentry/rules/history/preview.py

```diff
@@ -117,10 +121,13 @@ def categorize_conditions(conditions: Conditions) -> Tuple[Conditions, Condition
     issue_state_conditions = set()
     frequency_conditions = []
     for condition in conditions:
-        if condition["id"] in ISSUE_STATE_CONDITIONS:
-            issue_state_conditions.add(condition["id"])
-        else:
+        condition_id = condition["id"]
+        if condition_id in ISSUE_STATE_CONDITIONS:
+            issue_state_conditions.add(condition_id)
+        elif condition_id in FREQUENCY_CONDITIONS:
             frequency_conditions.append(condition)
+        else:
+            return [], []
```

Comment:
do we want to return here or just ignore it? I guess what would another unknown condition look like? Or if we add a new one that isn't hardcoded here?

would this behave correctly or incorrectly here?

Reply:
if we come up with another condition it'll just ignore it which might generate a misleading preview, if we return here it'll just return no preview immediately

Reply:
sweet makes sense. I can't think through it myself just wanted to make sure we were on the same page in terms of expected behavior